### PR TITLE
Add `ftdi` feature

### LINKED ISSUE #377: Enable ftdi feature in probe-rs
Please, can you enable the `ftdi` feature for `probe-rs` dependency in order to be able to use also FTDI debuggers?
Thanks

### PR DESCRIPTION
**Pull request by Urhengulas**
To test it install `probe-run` like this:

```console
$ cargo install --git https://github.com/knurling-rs/probe-run --branch ftdi --features ftdi
```

Fixes #377

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+- [#xxx] Add feature `tfdi` to enable `tfdi` debuggers
```
Comment (jamesmunns) [inline CHANGELOG.md:9]:
```suggestion
- [#xxx] Add feature `ftdi` to enable `ftdi` debuggers
```

Comment (Urhengulas) [reply CHANGELOG.md:9]: Oh wow. Thank you for catching that 😵‍💫

Comment (Urhengulas) [reply CHANGELOG.md]: Are you using a ftdi debugger and tested it by any chance?

Comment (jamesmunns) [reply CHANGELOG.md]: Nope. Was just reporting #382 and browsed around. I'm familiar with FTDI chips, so the naming mismatch was pretty visible to me.

Comment (Urhengulas) [reply CHANGELOG.md]: 👍🏾

**Pull request merged**